<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tiss</class>
 <widget class="QDialog" name="Tiss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>253</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>544</width>
    <height>253</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>253</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tiss</string>
  </property>
  <widget class="QPushButton" name="btnRun">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>220</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run!</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>523</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QSpinBox" name="spbLatRes">
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QSpinBox" name="spbLongRes">
      <property name="value">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLabel" name="lblCircleSeg">
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Circle segments</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spbLatMax">
      <property name="minimum">
       <number>-89</number>
      </property>
      <property name="maximum">
       <number>89</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Radius</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="lblLatMin">
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Min. Lat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QSpinBox" name="spbLineSeg">
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="lblLongMax">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Max. Long.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="spbLatMin">
      <property name="minimum">
       <number>-89</number>
      </property>
      <property name="maximum">
       <number>89</number>
      </property>
      <property name="value">
       <number>-60</number>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QDoubleSpinBox" name="spbRadius">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="value">
       <double>800.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblRadius">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Latitude resolution: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QLabel" name="lblCircleSeg_2">
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Line segments</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblLongMin">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Min Long.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="spbLongMax">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>150</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="lblLatMax">
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Max. Lat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QSpinBox" name="spbCircleSeg">
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spbLongMin">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>-150</number>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Longitude resolution:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
